src/types.ts

```diff
@@ -12,6 +19,21 @@ export interface ElysiaSwaggerConfig<Path extends string = '/swagger'> {
         | 'x-express-openapi-additional-middleware'
         | 'x-express-openapi-validation-strict'
     >
+    /**
+     * Choose your provider, Scalar or Swagger UI
```

Comment:
jsdoc for `scalarVersion` and `provider` are swapped I believe

Reply:
great catch! just fixed :)